tests/POST_request_JSON.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This is a JSON file, not a unit test.

Your test should be modeled after the testing done in the `cs334practice` repo.

Reply:
which folder should I put JSON file?